docs/pages/blog/mui-x-v7-beta.md

```diff
@@ -0,0 +1,245 @@
+  defaultExpandIcon={<ChevronRightIcon />}
+/>
+```
+
+Check out the [new component](https://next.mui.com/x/react-tree-view/rich-tree-view/items/)!
+
+## Charts
+
+Since the initial stable release of MUI X Chart a few months ago, we have been diligently listening to your feedback and focusing on refining the foundational charts to ensure they effectively cater to your diverse use cases.
+This includes bug fixes, doc improvements, and support for more complex scenarios.
+
+### Reference line
+
+The `<ChartsReferenceLine />` component enhances data visualization, providing users with a clear reference to better understand and analyze key data points.
+
+<img alt="Charts reference line" src="/static/blog/mui-x-v7-beta/charts-reference-line.png" width="1200" height="840" loading="lazy" style="margin-bottom: 16px;" />
+
+You can find more details on its [documentation](https://next.mui.com/x/react-charts/axis/#reference-line).
+
+## Date Pickers
+
+The focus has been mostly on developing the highly anticipated new components, DateTimeRangePicker and TimeRangePicker, alongside a significant effort to enhance accessibility in the fields, ensuring that our components are as inclusive and user-friendly as possible.
+
+While our journey towards these new additions continues, our commitment to improving existing features remains a top priority.
+Accordingly, we consistently update existing components to align seamlessly with the latest versions of supported date libraries.
+
+### Date Time Range Picker [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+Support new use cases with this most anticipated component.
+The latest addition to the Date Pickers suite is the `<DateTimeRangePicker />`, a component designed to enrich the user experience with advanced date and time selection.
+This intuitive picker simplifies the process of selecting date and time ranges, making it ideal for applications that require detailed scheduling or period selection.
+
+<video preload="metadata" style="margin-bottom: 10px;" autoplay muted loop playsinline width="600" height="508">
+  <source src="/static/blog/mui-x-v7-beta/date-time-range-picker.mp4" type="video/mp4">
+</video>
+
+Learn how to use and customize the [new component](https://next.mui.com/x/react-date-pickers/date-time-range-picker/) now!
+
+### Support for date-fns v3
+
+The Pickers now support both versions of v2 and v3 of date-fns and you can select your desired version from one of the two import lines:
+
+```jsx
+// date-fns v2.x
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+```
+
+```jsx
+// date-fns v3.x
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFnsV3';
+```
+
+## What's next
+
+As we approach the stable release of v7, our roadmap is well-defined, focusing on delivering key enhancements and features either just before or within a couple of weeks following the stable version's launch.
+
+**Data Grid**
```

Comment:
Is there a benefit to do bold text instead of an h3 here?

Reply:
My thought was to avoid any issue with the anchoring links since each component has two anchors with the same name, one for what's new and one for what's next.